modules/twinkleprotect.js

```diff
@@ -258,7 +258,7 @@ Twinkle.protect.callback.showLogAndCurrentProtectInfo = function twinkleprotectC
 			if (settings.expiry === 'infinity') {
 				protectionNode.push(' (indefinite) ');
 			} else {
-				protectionNode.push(' (expires ' + new Date(settings.expiry).toUTCString() + ') ');
+				protectionNode.push(' (expires ' + new Morebits.date(settings.expiry).calendar('utc') + ') ');
```

Comment:
I've done `Morebits.date.calendar('utc')` to match what's present in the block display and morebits `confirms`s, but I actually think that's not necessarily great, mainly because it doesn't *indicate* the UTC-ness and can be confusing for upcoming expirations.

```suggestion
				protectionNode.push(' (expires ' + new Morebits.date(settings.expiry).calendar() + ') ');
```

might be better, or even something with `.format`